MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,566 @@
+    {
+        ToolbarItems = new List<IImageEditorToolbarItem>()
+        {
+           new ImageEditorToolbarItem(){ Name = "Rectangle"},
+           new ImageEditorToolbarItem(){ Name = "Pen"},
+           new ImageEditorToolbarItem(){ Name = "AddText"}
+        },
+    }
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Add an item with the existing toolbar.
+
+You can explicitly add the `ImageEditorToolbarItem` to the `ImageEditorToolbar` manually.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+Image image = new Image();
+image.Source = ImageSource.FromResource("MyProject.Resources.Images.share.png");
+image.WidthRequest = 48;
+image.HeightRequest = 48;
+ImageEditorToolbar headerToolbar = imageEditor.Toolbars[0];
+ImageEditorToolbarGroupItem saveGroup = (ImageEditorToolbarGroupItem)headerToolbar.ToolbarItems[2];
+saveGroup.Items.Add(new ImageEditorToolbarItem() { Name = "Share", View = image });
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Orientation
+
+The toolbar orientation can be changed to achieve the desired design by using the `Orientation` property in the `ImageEditorToolbar` class. The default value is `ToolbarOrientation.Horizontal`.
+
+For example, you can set the toolbar to a vertical orientation:
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+footerToolbar.Orientaion = ToolbarOrientation.Vertical;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar Position
+
+The toolbar can be positioned before or after the image of the ImageEditor by using the `Position` property in the `ImageEditorToolbar` class. The default value is `ToolbarPosition.Start`.
+
+{% tabs %}
+
+{% highlight C# %}
+
+SfImageEditor imageEditor = new SfImageEditor();
+imageEditor.Source = ImageSource.FromFile("image.png");
+ImageEditorToolbar footerToolbar = imageEditor.Toolbars[1];
+footerToolbar.Position = ToolbarPosition.Start;
+    
+{% endhighlight %}
+
+{% endtabs %}
+
+## Toolbar size customization
+
+The size of the toolbar can be adjusted by using the `Size` property in the `ImageEditorToolbar` class.
+
+For horizontal orientation, the size will be considered as width, and for vertical orientation, it will be considered as height.
+
+{% highlight C# %}
+
+    SfImageEditor imageEditor = new SfImageEditor();
+    imageEditor.Source = ImageSource.FromFile("image.png");
+    imageEditor.ToolbarSettings.Background = Brush.LightGreen;
+    imageEditor.Toolbars[0].Size = 80;
+    imageEditor.Toolbars[1].Size = 50;
+
+{% endhighlight %}
+
+## Appearance customization
+
+You can customize the appearance of the toolbar by using the `ToolbarSettings` property. Below are some of the properties you can customize:
+
+* `Background`: Specifies background color of the toolbar.
```

Comment:
* `Background`: Specifies the background color of the toolbar.

Reply:
Updated